modules/tinymce/src/core/main/ts/keyboard/NavigationUtils.ts

```diff
@@ -69,6 +75,9 @@ const moveHorizontally = (editor: Editor, direction: HDirection, range: Range, i
 
 const moveVertically = (editor: Editor, direction: LineWalker.VDirection, range: Range, isBefore: (caretPosition: CaretPosition) => boolean,
                         isAfter: (caretPosition: CaretPosition) => boolean, isElement: (node: Node) => node is Element): Optional<Range> => {
+  if (!range.collapsed) {
```

Comment:
Is there a reason we're doing this in the vertical case and not the horizontal case? This is just a code smell sorry as it seems like an odd change.

Reply:
Fixed